dm_pix/_src/metrics_test.py

```diff
@@ -44,17 +44,29 @@ def setUp(self):
 
   @chex.all_variants
   def test_psnr_match(self):
-    psnr = self.variant(metrics.psnr)
+    psnr = self.variant(functools.partial(metrics.psnr, ignore_nans=False))
```

Comment:
This partial should not be needed (maybe this is now a consequence of your last commit).
Can you please check it? Same in line 60 below.

Reply:
Done!